### PR TITLE
feat(ui): redesign sign in and registration

### PR DESCRIPTION
**Pull request by myyxl**
### 📚 Description

The MR changes the design of the register and sign in flow, to make it more appealing. 

### 📝 Checklist

- [x] All commits follow the Conventional Commit format
- [x] The PR's title follows the Conventional Commit format

### CODE REVIEW

**Comment by myyxl** [comment]
Not sure why I can't set the width of the input field in the password reset form. Can't get it to work, maybe you can take a look.

**Review by dargmuesli — COMMENTED** [review]
Some suggestions on commit messages, just fyi, won't make the build fail:
- try not to use upper case characters
- use dashes in the scope (`fix(some-example)` instead of `fix(someExample)`)
- the scope should most likely not require multiple things (`feat(somthing)` instead of `(this-and-that)`)
- the part after the colon should most likely not repeat the scope (e.g. `fix(modals): correct popups` instead of `fix(popups): correct popups`)

File: src/components/layout/LayoutPageTitle.vue [hunk 1]
```diff
@@ -35,10 +42,12 @@ export interface Props<T extends RoutesNamesList, P extends string> {
   isButtonEventCreateShown?: boolean
   title: string
   to?: NuxtRoute<T, P>
+  centered?: boolean
```
Comment (dargmuesli) [inline src/components/layout/LayoutPageTitle.vue:45]: Boolean properties should have the `is` prefix by convention.
Also, right now all properties in this codebase are sorted (or should be), so I'd go ahead and move this new property a few lines up.

File: src/components/form/input/FormInput.vue [hunk 2]
```diff
@@ -6,11 +6,11 @@
         'form-input-warning': warning,
```
Comment (dargmuesli) [inline src/components/form/input/FormInput.vue:6]: Having the changes below does not show form input errors anymore:
image
See that `❌ Required` is missing. On master:
image

File: src/components/form/input/FormInputCaptcha.vue [hunk 3]
```diff
@@ -12,6 +12,7 @@
       :options="{
         'error-callback': () => (isLoading = false),
         'expired-callback': () => emit('input', undefined),
+        theme: themeColor,
```
Comment (dargmuesli) [inline src/components/form/input/FormInputCaptcha.vue:15]: TODO: have this be reactive

Comment (dargmuesli) [reply src/components/form/input/FormInputCaptcha.vue:16]: Done in e5912539f3670dcd2977318b411ba12ffde8926b, please verify :raised_hands:

File: src/components/form/input/FormInput.vue (hunk 2 above)
Comment (myyxl) [reply src/components/form/input/FormInput.vue:6]: If you remove the required property the warning will also disappear. Maybe we can split it up? Or we can just ignore it for now and keep the required property.

Comment (dargmuesli) [reply src/components/form/input/FormInput.vue:6]: Huh, cannot reproduce anymore? Sus. But will resolve for now.

**Pull request merged**